src/components/common/EmulatorDisabled.tsx

```diff
@@ -59,6 +59,43 @@ export const EmulatorDisabled: React.FC<
   </>
 );
 
+export const ExtensionsEmulatorDisabled: React.FC<
+  React.PropsWithChildren<{}>
+> = () => (
+  <>
+    <GridCell desktop={2} tablet={0} phone={0}>
+      {/* Spacing */}
+    </GridCell>
+    <GridCell span={8} align="middle">
+      <Card className="EmulatorDisabled">
+        <div className="EmulatorDisabled-graphics">
+          <img src="/assets/extensions/default-extension.png" alt="" />
+        </div>
+        <div className="EmulatorDisabled-description">
+          <Typography use="headline6" tag="h3">
+            The Extensions Emulator is currently not running.
+          </Typography>
+          <Typography use="body1" tag="p">
+            If you are not using Extensions in your testing, everything is fine.
+            If you want to use Extensions in your testing, add them to your
+            extension manifest. To do so, either to add a single extension
+            instance by running:
+            <code>firebase ext:install &gt;publisherId/extensionId&lt;</code>
+            or export the extensions instances from a production project by
+            running
+            <code>firebase ext:export --project=your-project</code>.
+          </Typography>
+          <Typography use="body1" tag="p">
+            <a href="https://firebase.google.com/docs/extensions/manifest">
```

Comment:
There is a docs link component somewhere which takes care of the looks

Reply:
Removed this to inline it instead